webapp/src/views/home/home.vue

```diff
@@ -16,72 +16,91 @@
         <div class="my-3 font-weight-bold">
           <h4 class="text-primary">Some stats</h4>
           <b-row class="text-center shadow bg-white rounded px-4 pt-4 pb-3">
-            <b-col sm="12" md="4">
+            <b-col sm="12" md="3">
               <p class="text-secondary display-4 mb-0">{{stats ? formatWithCommaSeparator(stats.numContributors) : 0}}</p>
               <p>Contributors so far</p>
             </b-col>
-            <b-col sm="12" md="4">
+            <b-col sm="12" md="3">
               <p class="text-primary display-4 mb-0">{{stats ? (formatWithCommaSeparator(stats.totalContributed) + '+') : 0}}</p>
-              <p>Money shared so far</p>
+              <p>Total amount contributed</p>
             </b-col>
-            <b-col sm="12" md="4">
+            <b-col sm="12" md="3">
+              <p class="text-primary display-4 mb-0">{{stats ? (formatWithCommaSeparator(stats.totalDistributed) + '+') : 0}}</p>
+              <p>Total amount distributed</p>
+            </b-col>
+            <b-col sm="12" md="3">
               <p class="text-primary display-4 mb-0">{{stats ? formatWithCommaSeparator(stats.numBeneficiaries) : 0}}</p>
-              <p>Beneficiaries so far</p>
+              <p>Recipients so far</p>
             </b-col>
           </b-row>
         </div>
       </section>
 
-      <section class="my-5 pt-5 text-center" id="beneficiaries">
-        <h1 class="text-primary mb-5">The Beneficiaries</h1>
+      <section class="my-5 pt-5 text-center" id="how-it-works">
         <p>
-          All beneficiaries get <span class="text-secondary font-weight-bold">Ksh 2,000</span> 
-          to get basic supplies during this trying period. <br/>
-          Here are some of the people whom your contribution will go along to help.
+          Social Relief aims to send <span class="text-secondary font-weight-bold">Ksh 2,000</span> 
+          per month to each beneficiary on the platform between September and November 2020.<br/>
+          This amount will help those in need address their basic needs and get back on their feet following
```

Comment:
Put a `to` between `need` and `address`, as in `This amount will help those in need to address...`